pkgs/applications/science/robotics/mujoco/default.nix

```diff
@@ -49,24 +49,24 @@ let
   qhull = fetchFromGitHub {
     owner = "qhull";
     repo = "qhull";
-    rev = "3df027b91202cf179f3fba3c46eebe65bbac3790";
-    hash = "sha256-aHO5n9Y35C7/zb3surfMyjyMjo109DoZnkozhiAKpYQ=";
+    rev = "0c8fc90d2037588024d9964515c1e684f6007ecc";
+    hash = "sha256-Ptzxad3ewmKJbbcmrBT+os4b4SR976zlCG9F0nq0x94=";
   };
   tinyobjloader = fetchFromGitHub {
     owner = "tinyobjloader";
     repo = "tinyobjloader";
     rev = "1421a10d6ed9742f5b2c1766d22faa6cfbc56248";
-    hash = "sha256-9z2Ne/WPCiXkQpT8Cun/pSGUwgClYH+kQ6Dx1JvW6w0=";
```

Comment:
why this diff? if unnecessary, let's not change so as to keep the git blame clean

Reply:
Oh yes, my mistake !